superduperdb/base/config.py

```diff
@@ -53,7 +53,7 @@ class Cluster(JSONable):
     distributed: bool = False
     deserializers: t.List[str] = Factory(list)
     serializers: t.List[str] = Factory(list)
-    dask_scheduler: str = 'tcp://localhost:8786'
+    dask_scheduler: str = 'tcp://scheduler:8786'
```

Comment:
Please explain.

Reply:
Instead of addressing the scheduler implicitly via `localhost`, we use the actual name of the `scheduler`. Similarly, we do not access mongo via `mongodb://user:pass@localhost:27017` but as `mongodb://user:pass@mongodb:27017`.

This makes the whole thing more realistic, and easier to connect to external dask schedulers.

Reply:
Gotcha.